cvat/apps/dataset_manager/bindings.py

```diff
@@ -1664,6 +1668,33 @@ def GetCVATDataExtractor(
 class CvatImportError(Exception):
     pass
 
+class CvatDatasetNotFoundError(Exception):
+    message = field(default="")
+    reason = field(default="")
+    format_name = field(default="")
+    _docs_base_url = f"{settings.CVAT_DOCS_URL}/manual/advanced/formats/"
```

Comment:
If this class is not declared with `@dataclass` or `@attrs.define`, these fields will be class members, not instance fields.